mindmap/mindmap.py

```diff
@@ -254,18 +227,13 @@ def get_current_mind_map(self, path_prefix: str) -> dict | None:
             dict: The current mind map content.
             None: If the file does not exist.
         """
-        s3_client, aws_bucket_name = self.connect_to_s3()
+        mindmap_storage = get_mindmap_storage()
```

Comment:
Only load this once before the class definition in a variable `MINDMAP_STORAGE`

Reply:
I tried to load it that way (similar to the H5P xblock), but it didn't work 🤔

Reply:
Then define a class level property with that value

Reply:
That doesn't work for me either.

Reply:
Then set it as a Django setting or leave it like it if it doesn't work

Reply:
Adding a new Django setting I think is not necessary, I will leave it as it is now.